src/abstracts/Command.ts

```diff
@@ -5,11 +5,13 @@ abstract class Command {
 	private readonly name: string;
 	private readonly description: string;
 	private readonly options: CommandOptions | undefined;
+	private readonly aliases: Array<string>;
```

Comment:
Can you use `string[]` over `Array<string>` please, it's just shorter and neater in my opinion?

Reply:
Nice, I didn't know about that syntax, it is cleaner indeed. I'm going to fix it :+1: